setup-rhmap.sh

```diff
@@ -92,7 +90,7 @@ then
 fi
 
 # checkout the correct branch e.g. release-4.6.0-rc1
-echo "enter branch/tag name e.g. release-4.6.0-rc1:"
+echo "enter branch/tag name e.g. release-4.6.0-rc1 or FH-v4.6"
```

Comment:
Will the release-4.6.0-rc1 work?

Reply:
no, a fair point will change to just the 4.6 branch